szn-select--ui.js

```diff
@@ -201,6 +202,46 @@
     if (suggestedHeight !== currentHeight) {
       instance._dropdownContent.style.height = `${suggestedHeight}px`
     }
+
+    const buttonWidth = instance._button.offsetWidth
+    const dropDownCssMinWidth = getElementCssMinWidth(instance._dropdownOptions)
+
+    if (dropDownCssMinWidth < buttonWidth) {
+      instance._dropdownContent.style.minWidth = `${buttonWidth}px`
+    }
+  }
+
+  function getElementCssMinWidth(elm) {
```

Comment:
Musím uznať, že je to pekné riešenie. Otázka znie, či nie je zbytočne silné (plus prehliadol si ako jednotky %). Jediné jednotky, ktoré sú závislé na kontexte, sú em a %, Ostatné (px, rem, pt, vw, vh) sú nezávislé na kontexte použitia. Čo keby sme teda neprepočítavali veľkosť na px a len v dokumentácii uviedli, že nie je doporučované kvôli týmto dôvodom používať jednoty em a %?

Reply:
prepocet na px je nutny kvuli porovnani s sirkou selecu. Doplnil jsem i zbyvajici jednotky. K tomu nastaveni bych v dalsim kroku doplnil do README zminku o dropdownAutoStretch a k nastaveni --szn-select---min-width resp. --szn-select--ui--dropdown-min-width, ze pro dropdown se doporucuje nepouzivat jednotky EM a %, protoze nelze zajistit spravne zobrazeni a to i v pripade, kdzy nebude podporovan dropdownAutoStretch.